pyanaconda/modules/payloads/source/nfs/nfs.py

```diff
@@ -42,6 +43,11 @@ def type(self):
         """Get type of this source."""
         return SourceType.NFS
 
+    @property
+    def description(self):
+        """Get description of this source."""
+        return _("NFS server %s") % self.url
```

Comment:
Please, use `format`.

Reply:
The reason I went with % was to preserve the same strings for translation. Is that a non-priority? (Just asking - got no idea.)

Reply:
I see. It would be a priority during a string freeze, otherwise it doesn't matter.

Reply:
Makes sense, let's change it then.